cases/pysb/0001/_0001.yaml

```diff
@@ -0,0 +1,11 @@
+format_version: 1
+parameter_file: _parameters.tsv
+problems:
+- condition_files:
+  - _conditions.tsv
+  measurement_files:
+  - _measurements.tsv
+  observable_files:
+  - _observables.tsv
+  sbml_files:
```

Comment:
Shall we change that to `pysb_files` directly?

Reply:
probably makes sense.

Reply:
Would leave as is for now, so the yaml is still syntactically correct for the current PEtab. Definitely needs changing in the future, but it needs to be discussed how that would look like, but I would avoid changing things twice.